facia-tool/app/controllers/FrontendDependentController.scala

```diff
@@ -0,0 +1,70 @@
+package controllers
+
+import play.api.mvc._
+import play.api.libs.json.{JsNull, Json, JsString, JsValue}
+import play.api.Play
+import play.api.Play.current
+import model.Cached
+import auth.ExpiringActions
+import slices.{FixedContainers, DynamicContainers, ContainerJsonConfig}
+import conf.{Configuration, FaciaToolConfiguration}
+import common.Edition
+
+
+object Defaults {
+  implicit val jsonFormat = Json.format[Defaults]
+}
+
+case class Defaults(
+  dev: Boolean,
+  env: String,
+  editions: Seq[String],
+  navSections: Seq[String],
+  email: String,
+  avatarUrl: Option[String],
+  lowFrequency: Int,
+  highFrequency: Int,
+  standardFrequency: Int,
+  sentryPublicDSN: Option[String],
+  fixedContainers: Seq[ContainerJsonConfig],
+  dynamicContainers: Seq[ContainerJsonConfig]
+)
+
+object FrontendDependentController extends Controller {
```

Comment:
not 100% sure the naming of this class is right? should be FrontsConfigController maybe?